doc/conf_ref.rst

```diff
@@ -336,7 +336,7 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     Set this to True to disable module RPM filtering and make all RPMs from the repository available. The default is False.
     This allows user to create a repository with cherry-picked hotfixes that are included in a package set on a modular system.
 
-.. _mirrorlist-label:
```

Comment:
Please move the ```.. _mirrorlist-label:``` to correct place. It is needed by line 43 ```or :ref:`mirrorlist <mirrorlist-label>` option definition.```

Reply:
Ther is one on the line 324. There were two definitions of mirrorlist-label, I've only removed the incorrect one.

Reply:
Sorry, my search engine in PyCharme simply ignore that line